docs/UserGuide.md

```diff
@@ -167,14 +167,14 @@ Format: ‘find n/NAME p/PHONE_NUMBER r/ROOM_NUMBER t/Tags’’
 * Persons matching at least one keyword will be returned (i.e. `OR` search).
   e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
 * Only full words will be matched for room_number and phone_number, the arguments
-  should be exactly the same as the person's room number and phone number
-* order of the tags doesn't matter `t/friends colleagues` works the same
+  should be exactly the same as the person's ROOM_NUMBER and PHONE_NUMBER
+* Order of the tags do not matter `t/friends colleagues` works the same
   as `t/colleagues friends`
 
 Examples:
-* `find p/test_number n/test_name r/08-1234 t/friends colleagues`  
+* `find p/94509592 n/Alex Jones r/08-1234 t/friends colleagues`  
```

Comment:
The find here for t/friends colleagues only works if the two words are in one tag as "friends colleagues" but doesn't work if the person has two tags, "friends" and "colleagues". Should this be written into the UG or fixed for tags?